src/redux/watchList/reducer.js

```diff
@@ -54,7 +54,18 @@ export default (state = initalState, action) => {
       return {
         ...state,
         counter: state.counter + 1,
-        watchlists: [...state.watchlists, { ...action.payLoad, id: state.counter }],
+        watchlists: [
+          ...state.watchlists,
+          {
+            id: state.counter + 1,
+            name: action.payLoad,
+            stocks: [],
+          }],
+      };
+    case DELETE_WATCHLIST:
+      return {
+        counter: state.counter - 1,
```

Comment:
i think we shouldn't chnage the counter, just like in the dabase it is an id, and we wont assign the same id for two different watchlists